### PR TITLE
Allow customers to override Journeys CTA links

### PR DESCRIPTION
**Pull request by jwangler-branch**
* add journey CTA replacement variable ($journeys_cta)
* users are able to replace the journeys link by: `setBranchViewData({"$journeys_cta": "http://my.own.link"})`

### CODE REVIEW

**Comment by rubinsingh** [comment]
👍

File: src/journeys_utils.js [hunk 1]
```diff
@@ -908,3 +908,47 @@ journeys_utils.setJourneyLinkData = function(linkData) {
...
+
+	if(!journeys_utils.branch._branchViewData.data){
+		return false;
+	}
+
+	return journeys_utils.branch._branchViewData.data[key];
+};
+
+journeys_utils.getBranchViewDataItemOrUndefined = function(name){
+	if(journeys_utils.getValueForKeyInBranchViewData(name)){
```
Comment (aaaronlopez) [inline src/journeys_utils.js]: hey @jwangler-branch , can you fix the spacing here? There is a linter in the package.json.

Just run `npm install` and it should pick up the hook going forward

Comment (aaaronlopez) [reply src/journeys_utils.js]: for example: `if (journeys_utils.getValueForKeyInBranchViewData(name)) {`

similar patterns in the rest of the code

Comment (jwangler-branch) [reply src/journeys_utils.js]: Yes, I'll set lint as auto-execute.. :)

**Pull request merged**